### PR TITLE
Revert NUE

### PR DESCRIPTION
**Pull request by lukicdarkoo**
With Webots R2020b we cannot do a proper conversion of ROS worlds to ENU (we need to introduce breaking changes).

ENU conversion will be available in `develop` branch, compatible with Webots R2021a (started https://github.com/cyberbotics/webots_ros/pull/38).

### CODE REVIEW

File: worlds/keyboard_teleop.wbt [hunk 1]
```diff
@@ -1,35 +1,30 @@
...
+  translation 0.133486 0.3 -1.17773
   name "cardboard box(1)"
 }
 WoodenBox {
-  translation -0.797045 -1.02408 0.3
-  rotation 1 0 0 1.5707963267948966
+  translation -0.797045 0.3 1.02408
 }
 DEF PIONEER_3DX Pioneer3dx {
-  translation 0 0.0975 0.1
```
Comment (omichel) [inline worlds/keyboard_teleop.wbt:32]: It seems we are missing a translation here, isn't it?

File: worlds/panoramic_view_recorder.wbt [hunk 2]
```diff
@@ -1,24 +1,21 @@
...
 Robot {
-  rotation 1 0 0 1.5707963267948966
   controller "ros"
   supervisor TRUE
 }
 Floor {
-  rotation 1 0 0 1.5707963267948966
 }
 Nao {
-  translation 0 0.334 0.33
```
Comment (omichel) [inline worlds/panoramic_view_recorder.wbt:21]: Aren't we missing a translation here?

**Review by omichel — CHANGES_REQUESTED** [review]
Why did you modified the .wbproj files? It seems they should be affected by the change to ENU...

**Comment by lukicdarkoo** [comment]
It should be equal to 4f3c3d0e949e17b:
https://github.com/cyberbotics/webots_ros/compare/revert-nue..4f3c3d0e949e17b2be7948385ce06d08bd1cf351

**Review by omichel — APPROVED** [review]
All right.
I didn't understood initially, that this PR was reverting previous changes...

**Pull request merged**